forge/src/lib.rs

```diff
@@ -47,8 +47,32 @@ pub mod test_helpers {
         Project::builder().paths(paths).ephemeral().no_artifacts().build().unwrap()
     });
 
+    pub static LIBS_PROJECT: Lazy<Project> = Lazy::new(|| {
+        let paths = ProjectPathsConfig::builder()
+            .root("../testdata")
+            .sources("../testdata")
+            .build()
+            .unwrap();
+
+        let mut settings = Settings::default();
+        let libs = // change this
```

Comment:
Is the comment still relevant?

Reply:
No, as far as I know this is the correct way to link to a predeployed library